packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubbleContent.base.tsx

```diff
@@ -113,11 +114,15 @@ export class TeachingBubbleContentBase extends BaseComponent<ITeachingBubbleProp
 
     if (primaryButtonProps || secondaryButtonProps || customFooterContent) {
       footerContent = (
-        <div className={classNames.footer}>
-          {primaryButtonProps && <PrimaryButton {...primaryButtonProps} className={classNames.primaryButton} />}
-          {secondaryButtonProps && <DefaultButton {...secondaryButtonProps} className={classNames.secondaryButton} />}
-          {customFooterContent && <span>{customFooterContent}</span>}
-        </div>
+        <Stack horizontal horizontalAlign={customFooterContent ? 'space-between' : 'end'}>
+          <Stack.Item className={classNames.footer} align="center">
+            {<span>{customFooterContent}</span>}
+          </Stack.Item>
+          <Stack.Item>
```

Comment:
This is a pretty big DOM change, and will probably break people that are doing customizations on the footer thinking it covered the buttons as well. Can the footer be on the above stack?

Either way, we need to communicate this change out via Teams and the fabricreact alias so that people know this DOM change is coming.